tests/unit/statistical_testing/KernelSelection_unittest.cc

```diff
@@ -245,8 +245,8 @@ TEST(KernelSelectionMaxTestPower, quadratic_time_single_kernel)
 
 	auto gen_p=new CMeanShiftDataGenerator(0, dim, 0);
 	auto gen_q=new CMeanShiftDataGenerator(difference, dim, 0);
-	gen_p->CSGObject::put("seed", seed);
-	gen_q->CSGObject::put("seed", seed);
+	gen_p->put("seed", seed);
```

Comment:
not sure I get why this is part of the PR?

Reply:
an artifact from the random mixin pr because of the overloaded function